geode-gfsh/src/main/java/org/apache/geode/management/internal/cli/functions/RegionCreateFunction.java

```diff
@@ -129,6 +133,16 @@ private CliFunctionResult handleException(final String memberNameOrId, final Str
     return new CliFunctionResult(memberNameOrId, CliFunctionResult.StatusState.ERROR);
   }
 
+  private void waitUntilConditionIsMet(BooleanSupplier awaitedCondition, int timeoutInMs) {
```

Comment:
This can be done with GeodeAwaitility. No need for a new function.

Reply:
I think that GeodeAwaitility is used only for the test (as states in the description).

```
/**
 * Utility to set consistent defaults for {@link org.awaitility.Awaitility} calls for all Geode
 * tests.
 */
public class GeodeAwaitility {

```
Also, I cannot find any usage instead of tests.

Reply:
Good point. That is a shame.